src/apps/backend/modules/account/rest_api/account_view.py

```diff
@@ -3,32 +3,27 @@
 from flask import request, jsonify
 from flask.views import MethodView
 from flask.typing import ResponseReturnValue
-from modules.account.types import AccountSearchByIdParams, CreateAccountParams
+
+from modules.account.types import AccountSearchByIdParams, CreateAccountParams, ResetPasswordParams
 from modules.account.account_service import AccountService
-from modules.account.errors import AccountNotFoundError, AccountWithUserNameExistsError
 
 class AccountView(MethodView):
   def post(self) -> ResponseReturnValue:
-    try:
       request_data = request.get_json()
       account_params = CreateAccountParams(**request_data)
       account = AccountService.create_account(params=account_params)
       account_dict = asdict(account)
       return jsonify(account_dict), 201
-    except AccountWithUserNameExistsError as exc:
-      return jsonify({
-        "message": exc.message,
-        "code": exc.code,
-      }), 400
       
   def get(self, id) -> ResponseReturnValue:
-    try:
       account_params = AccountSearchByIdParams(id=id)
       account = AccountService.get_account_by_id(params=account_params)
       account_dict = asdict(account)
       return jsonify(account_dict), 200
-    except AccountNotFoundError as exc:
-      return jsonify({
-        "message": exc.message,
-        "code": exc.code,
-      }), 400
+
+  def patch(self, id) -> ResponseReturnValue:
```

Comment:
If this API function is doing reset, then why the name `patch`?
It would have made sense if we are updating attributes of the account here.

We should have a different function for resetting password.

Reply:
@bbbneo333  - Updating password can be considered as PATCH to current account object (assuming password is its attributes). I guided this somewhat, FYI, in the original boilerplate as well.

Is this the only outstanding comment pending?